app/migrations/fromGE/rds_migrate.py

```diff
@@ -0,0 +1,196 @@
+import MySQLdb
+import secrets as SECRETS
```

Comment:
I'm now put a secrets.py file containing the DB credentials. Should i share these using  Sops or are they to sensitive?